### DISCUSSION TITLE
Back-End - JS Frameworks/APIs

### DISCUSSION THREAD

**Discussion by JosephSuvak** [JosephSuvak/QR-Ticket-Generator#12] [General]
For Back-End conversations.

**Comment by JohnBanas** [comment]
### Changes

## New Routes

- There is a new login api in user routes to save/start a cookie function

### IF YOU WANT DATA TO GO TO HANDLEBARS

- Changed concert-router.js `get('/') Concert.findAll` to include all attributes. You **HAVE** to include any attributes from the model you want to display in handlebars. It's the way we have it set up. You cannot just put `.findAll({})` that route has been added and all concerts can be viewed from the user account page.
- There is a new logout user and destroy session api in user-routes.js
- There are new login and sign up api in homepage-routes.js
- Added api route to get all tickets and render to user account on login

**Comment by JohnBanas** [comment]
API routes added to get single concert and single ticket. Add ticket to a user's account and delete ticket as well.
